theme/style.css

```diff
@@ -15,4 +15,8 @@
   text-align: center;
 }
 
+.kevin {
```

Comment:
Make sure that you actually have some styles, and that you're calling out those styles by `id` instead of `class`!

Reply:
ok, i just updated the styles did you get the pull request?